dxw/dxw-members-only
# Update composer to create an installable set on modern instances

This commit updates the composer file to set the platform to be php 7.4.3 to allow the package set to be installed on modern machines running recent versions of php (eg php 8.2)

to test:
download the code locally and ensure that composer install completes as expected.

- [x] Version number has been bumped
